src/event_log.rs

```diff
@@ -4,8 +4,12 @@ use crate::config::Config;
 use crate::ptrace_control::*;
 #[cfg(ptrace_supported)]
 use crate::statemachine::ProcessInfo;
+#[cfg(ptrace_supported)]
 use crate::statemachine::TracerAction;
-use crate::traces::{Location, TraceMap};
+#[cfg(ptrace_supported)]
```

Comment:
I don't think this change is required? The TraceMap is still used for llvm coverage, it might trigger an unused warning though

Reply:
TraceMap is just below, with an unconditional import. And yes, this is my second commit, just fixing some unused warnings. I can take it out if you prefer.

Reply:
nah it's fine was just making sure